tfs/src/main/java/hudson/plugins/tfs/util/BuildWorkspaceConfigurationRetriever.java

```diff
@@ -45,17 +45,23 @@ public BuildWorkspaceConfiguration getLatestForNode(Node needleNode, Run<?,?> la
     
     public static class BuildWorkspaceConfiguration extends WorkspaceConfiguration {
         private static final long serialVersionUID = 1L;
-        private final AbstractBuild<?, ?> build;
+        private final transient AbstractBuild<?, ?> build;

```

Comment:
I agree the persisted Build instance is a bad thing in any case. But the one who solves that has to ensure the plugin's logic does not suffer. Rejuice methods or whatever. It is not enough to just make the field transient